src/content/docs/zh-cn/guides/configuring-astro.mdx

```diff
@@ -94,7 +94,6 @@ const { ...props } = Astro.props;
     <!-- 你的页面内容在这 -->
   </body>
 </html>
-```
```

Comment:
This change seems unintentional and like it might break the formatting.

Reply:
Thanks, some other changes got mixed up in here, because the translators are translating under our feet and it's so difficult to keep branches updated. All is fixed!